### PR TITLE
Update documentation for AuthComponent redirect changes.

### PR DESCRIPTION
**Pull request by markstory**
Refs cakephp/cakephp#9410

### CODE REVIEW

File: en/appendices/3-4-migration-guide.rst [hunk 1]
```diff
@@ -40,6 +42,10 @@ behavior that may effect your application:
 * ORM\Query results will not typecast aliased columns based on the original
   columns type. For example if you alias ``created`` to ``created_time`` you
   will now get a ``Time`` object back instead of a string.
+* ``AuthComponent`` now uses a query string to store the redirect URL when an
+  unauthenticated user is redirected to the login page. Previously, this redirect
+  was stored in the session. Using the query string allows for better
+  multi-browser experience.
```
Comment (dereuromark) [inline en/appendices/3-4-migration-guide.rst]: multi browser or multi-tab?

File: en/controllers/components/authentication.rst [hunk 2]
```diff
@@ -257,7 +257,7 @@ returned is as per following rules:
   the ``Auth.redirect`` session value was used.
 - If there is no query string/session value and there is a config
   ``loginRedirect``, the ``loginRedirect`` value is returned.
-- If there is no redirect value and no ``loginRedirect``, / is returned.
+- If there is no redirect value and no ``loginRedirect``, ``/`` is returned.
```
Comment (ravage84) [inline en/controllers/components/authentication.rst]: We almost made that change simultaneously! But you beat me a few seconds :laughing:

File: en/appendices/3-4-migration-guide.rst (hunk 1 above)
Comment (ravage84) [inline en/appendices/3-4-migration-guide.rst]: Both, I guess.

Comment (dereuromark) [inline en/appendices/3-4-migration-guide.rst]: Multi browser worked before ;) It was the issues with tabs in the same browser that killed everything.

**Pull request merged**